src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Car get(Long id);
```

Comment:
```suggestion
    Optional<Car> get(Long id);
```

Давай возвращать Optional в методах, где мы можем получить null

Reply:
Ok!)